### PR TITLE
Changed default verbosity level to INFO

### PR DESCRIPTION
**Pull request by VicenteMovai**
There were a lot of DEBUG logs about `Creating tcp connection (redis-master)` , etc..

- Changing the default verbosity will improve these unnecessary logs
- Merged the additional log of "\n" with the general log

- [ ] Create docs on how to easily change the verbosity level to see the DEBUG again

:warning: Beware that Robotics and CSAI nodes that depend on DEBUG logs will be impacted

### CODE REVIEW

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]
Tests failing with:

`FAILED tests/test_logger.py::test_logger - UnboundLocalError: local variable `

File: tests/test_logger.py [hunk 1]
```diff
@@ -17,25 +24,29 @@ def check_trackback(caplog, should_have_trackback):
...
+@patch('movai_core_shared.envvars.MOVAI_GENERAL_VERBOSITY_LEVEL', 'DEBUG') 
 def test_logger(caplog):
     """Test for the logger that will generate new logger with get_logger
     and test out all the different verbosity and check the logs that are correct
     """
-    caplog.set_level(logging.DEBUG, logger="test_logger")
+    # caplog.set_level(logging.DEBUG, logger="test_logger")
+    # aux_env_var = envvars.MOVAI_GENERAL_VERBOSITY_LEVEL
+    # envvars.MOVAI_GENERAL_VERBOSITY_LEVEL = "DEBUG"
+    # os.environ["MOVAI_GENERAL_VERBOSITY_LEVEL"] = "DEBUG"
```
Comment (andreparames) [inline tests/test_logger.py:35]: This can be deleted, no?

**Comment by duartecoelhomovai** [comment]
We fixed the duplication of logs

**Comment by brunomovai** [comment]
Should this be being handled like this ? just desabling a platform feature ( debug logs) 

We have debug logs in a lot of our Nodes, with parameters to enable/disable them on demand. 
Lossing all of that work , because the platform is not able to handle it's logs perpery doesn't seem the correct way to go. 

And having a workadound to be able to get my logs, and then take the platform garbage again , is not a proper way to have this. 

Not to mention that taking all those logs my change sginificantly the performance level; to the point that we are not able to have a comparable product in deployment.
- Run system on client without debug's - Runs a certain way
- Run the sysmte on the client wtih debug ( because need to debug 1 node) - runs differently 

@VicenteMovai @AlexFernandes-MOVAI @DanielGoncalves-MOVAI @cptn-ahab

**Review by AlexFernandes-MOVAI — APPROVED** [review]

**Review by cptn-ahab — CHANGES_REQUESTED** [review]
@VicenteMovai can you explain in plain language what this PR intends to do?

**Comment by duartecoelhomovai** [comment]
> Tests failing with:
> 
> `FAILED tests/test_logger.py::test_logger - UnboundLocalError: local variable `

resolved

**Comment by duartecoelhomovai** [comment]
> Should this be being handled like this ? just desabling a platform feature ( debug logs)
> 
> We have debug logs in a lot of our Nodes, with parameters to enable/disable them on demand. Lossing all of that work , because the platform is not able to handle it's logs perpery doesn't seem the correct way to go.
> 
> And having a workadound to be able to get my logs, and then take the platform garbage again , is not a proper way to have this.
> 
> Not to mention that taking all those logs my change sginificantly the performance level; to the point that we are not able to have a comparable product in deployment.
> 
>     * Run system on client without debug's - Runs a certain way
> 
>     * Run the sysmte on the client wtih debug ( because need to debug 1 node) - runs differently
> 
> 
> @VicenteMovai @AlexFernandes-MOVAI @DanielGoncalves-MOVAI @cptn-ahab

I believe you jumped to fast out of your seat xD
What we did is changing the default logs on the whole system to INFO. 
Since this is a product that goes to clients (including you) i don't believe its acceptable to have the default as DEBUG. Even INFO is questionable to be honest but its just a first step. 
This means the DEFAULT level will be set to a less verbose logging set, and then be able to be customized on the start of the robots to adapt to the running needs. 
Another thing that is important to mention is that there are logger levels for different use cases. Logging level for call backs, for the platform, etc.. All them should be configurable independently.  We will provide an image on how to do it.
If you guys have more questions let me know.
@cptn-ahab @brunomovai

**Comment by brunomovai** [comment]
@duartecoelhomovai 

Then maybe i'm misunderstanding the scope and description of this PR: 

1 - Debug Log in a python code from a callaback
> Current behaviour : Shows in the terminal as a debug log
2 - Info Log in a python code from a callaback
> Current behaviour : Shows in the terminal as a info log
3 - Debug Log from the platform ( e.g. creating tcp connection)
> Current behaviour : Shows in the terminal as a debug log
4 - Info log from the platform ( e.g. shutting down node)

What would be the behaviour after this PR for the 4 cases ? 

`Another thing that is important to mention is that there are logger levels for different use cases. Logging level for call backs, for the platform, etc.. All them should be configurable independently. We will provide an image on how to do it.`

I should have the need to touch something like that, unless there's something very wrong that requires an in-depth debug though the infrastruture. 
Otherwise, the default of the platform should be the one that coves well the needs ot ir's clients/users ( developers, integrators, deployers, and operators )

**Comment by cptn-ahab** [comment]
@duartecoelhomovai Currently as a user of the system I have 2 types of logs
- UI logs that can be at 3 levels: Error, Warning, Info
- Technical logs (i.e container logs) that can be 5 levels: Critical, Error, Warning, Info and debug

Question Does this PR auto publish all debug logs as info logs or does it crop debug logs so they do not show anywhere?

**Comment by cptn-ahab** [comment]
To add to what @brunomovai said, it seems you're solving the problem just technically and not practically. Or at least not with the user in mind

**Comment by duartecoelhomovai** [comment]
Kept call back execution level in debug.
Platform verbosity was set to INFO

**Review by duartecoelhomovai — APPROVED** [review]

File: movai_core_shared/envvars.py [hunk 2]
```diff
@@ -10,14 +10,27 @@
...
+    "FATAL": FATAL,
+}
+
+
 # Setting for logging verbosity levels
 # Will be set only once at startup
-MOVAI_STDOUT_VERBOSITY_LEVEL = int(os.getenv("MOVAI_STDOUT_VERBOSITY_LEVEL", str(DEBUG)))
-MOVAI_FLEET_LOGS_VERBOSITY_LEVEL = int(os.getenv("MOVAI_FLEET_LOGS_VERBOSITY_LEVEL", str(DEBUG)))
+MOVAI_STDOUT_VERBOSITY_LEVEL = DEBUG # Minimum log supported by handlers
+MOVAI_FLEET_LOGS_VERBOSITY_LEVEL = int( LOG_LEVELS.get(os.getenv("MOVAI_FLEET_LOGS_VERBOSITY_LEVEL", "DEBUG").upper().strip(), DEBUG)) # Verbosity level for influxdb logs
```
Comment (AlexFernandes-MOVAI) [inline movai_core_shared/envvars.py:28]: not sure to understand this complicated syntax, I see 2 things we have to support 
- retrocompatible passing of level as integers
- user friendly  passing of level as strings
+ handling of other strings
normally `getLevelName` is enough see example below:

```
import logging
import os

def get_log_level(env_var_name, default=logging.INFO):
    """ Get the log level from the environment variable `env_var` and return it as a logging level """
    level = os.getenv(env_var_name, default)  # Default to 'INFO' if not set

    try:
        # Try converting to an integer level
        level = int(level)
    except ValueError:
        # If it's not an integer, it must be a string level
        pass

    # handle failure of getLevelName
    ret_level = logging.getLevelName(level)
    if isinstance(ret_level, str) and ret_level.startswith('Level '):
        print('Invalid log level: %s' % ret_level)
        return default

    return ret_level

# Test
os.environ["TEST"]="WARNING"
print(get_log_level("TEST"))  # Output: 30 (WARNING)
logging.basicConfig(level=get_log_level("TEST"))

os.environ["TEST"]="10"
print(get_log_level("TEST"))  # Output: DEBUG
logging.basicConfig(level=get_log_level("TEST"))

os.environ["TEST"]="toto"
print(get_log_level("TEST"))  # Output: 20 (INFO)
logging.basicConfig(level=get_log_level("TEST"))
```

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]
try about simplification of syntax on the log level env vars

**Review by AlexFernandes-MOVAI — APPROVED** [review]

**Pull request merged**